static/js/redux/ui/modals/add_advisor_modal.jsx

```diff
@@ -0,0 +1,148 @@
+      lastName: '',
+      submitted: false,
+    };
+  }
+
+  componentDidMount() {
+    if (this.props.isVisible) {
+      this.modal.show();
+    }
+  }
+
+  componentDidUpdate() {
+    if (this.props.isVisible) {
+      this.modal.show();
+    }
+    if (this.state.submitted === true) {
+      window.location.reload();
+      this.setState({ submitted: !this.state.submitted });
+    }
+  }
+
+  sendJHED(event) {
+    this.setState({ jhed: event.target.value });
+  }
+
+  sendFirstName(event) {
+    this.setState({ firstName: event.target.value });
+  }
+
+  sendLastName(event) {
+    this.setState({ lastName: event.target.value });
+  }
+
+  submitContent() {
+    if (this.state.comment !== '') {
+      fetch(getAdvisorTestAdd(), {
+        method: 'POST',
+        headers: {
+          'X-CSRFToken': Cookie.get('csrftoken'),
+          accept: 'application/json',
+          'Content-Type': 'application/json',
+        },
+        body: JSON.stringify({
+          JhedId: this.state.jhed,
+          FullName: `${this.state.lastName},${this.state.firstName}`,
+        })
+      })
+        .then(() => this.setState({ jhed: this.state.jhed = '',
+          firsfirstNamet_name: this.state.firstName = '',
+          lastName: this.state.lastName = '',
+          submitted: !this.state.submitted }));
+    }
+  }
+
+  render() {
+    const { jhed, firstName, lastName } = this.state;
+
+    const modalHeader =
+      (<div className="modal-content">
+        <div className="modal-header">
+          <h1>Add New Advisors</h1>
+          <div className="modal-close" onClick={() => this.modal.hide()}>
+            <i className="fa fa-times" />
+          </div>
+        </div>
+      </div>);
+    const modalStyle = {
+      width: '100%',
+    };
+    return (
+      <Modal
+        ref={(c) => {
+          this.modal = c;
+        }}
+        className="SIS-import-data-modal abnb-modal max-modal"
+        modalStyle={modalStyle}
+        onHide={() => {
+          this.props.toggleAddAdvisorModal();
+        }}
+      >
+        {modalHeader}
+        <p><br /></p>
+        <form action="#1">
+          <textarea
+            className="cf-input"
+            rows="1" placeholder="Enter Advisor JHED"
+            value={jhed}
+            onChange={event => this.sendJHED(event)}
+          />
+          <textarea
+            className="cf-input"
+            rows="1" placeholder="Enter Advisor First Name"
+            value={firstName}
+            onChange={event => this.sendFirstName(event)}
+          />
+          <textarea
+            className="cf-input"
+            rows="1" placeholder="Enter Advisor Last Name"
+            value={lastName}
+            onChange={event => this.sendLastName(event)}
+          />
+          <p><br /></p>
+          <input
+            className="send-btn"
+            type="submit"
+            value="Send Info"
+            onClick={() => this.submitContent()}
+          />
+        </form>
```

Comment:
Add some CSS:
![image](https://user-images.githubusercontent.com/27985363/115654354-02aa2f80-a2ff-11eb-9043-c22209a9bb5e.png)

We're presenting with this modal so it has to be up to that standard.

Reply:
I would center the Send Info button, make it green, add some margins.
The textfields should expand so at least the text hints are complete (so there's no vertical scrollbar), I would make it extend all the way with some margins on the left and right. In other words, make it pretty